SPEC/CONFIG.md

```diff
@@ -14,7 +14,7 @@
 
 `key` is the key of the value that should be fetched from the config file. If no key is passed, then the whole config should be returned. `key` should be of type String.
 
-`callback` must follow `function (err, config) {}` signature, where `err` is an error if the operation was not successful and `config` is a JSON object containing the configuration of the IPFS node.
+`callback` must follow `function (err, config) {}` signature, where `err` is an error if the operation was not successful and `config` is an object containing the configuration of the IPFS node.
```

Comment:
Are you saying that the config is not pure JSON?

Reply:
The config is a JavaScript object, which can be stringified to JSON.

Please see https://github.com/ipfs/js-ipfs-api/issues/822 and https://github.com/ipfs/js-ipfs-api/pull/825 😄

Reply:
It should be an object everywhere. What we mean by it being a JSON object is that it should include __no buffers__ on it.

Reply:
@diasdavid Oh, makes sense then. I thought "JSON object" == "Buffer with JSON string". So this PR can be closed, but https://github.com/ipfs/js-ipfs-api/pull/825 should be merged then.